angelspetrochem/prototypes/items/petrochem-basics.lua

```diff
@@ -107,16 +107,16 @@ data:extend(
         nil,
       icons = angelsmods.functions.create_gas_fluid_icon(
         {"__angelspetrochem__/graphics/icons/molecules/carbon-dioxide.png", 72},
-        "ccc"
+        "CCO"

```

Comment:
Same as before, leave as CCC to match fluid and recipe icon, if you want to make a difference between monoxide and dioxide, you have to change both icons so that they still match...